Android/plugin-manifest.json

```diff
@@ -56,5 +56,6 @@
             "tooltip_text": "The text on the button that appears alongside a connectivity error message",
             "default": "Try Again"
         }
-    ]
+    ],
+    "about": "*Built with love by the Applicaster Owned Plugins Team*  \n**Supports:** *iOS, Android*  \n*Current Version: 1.1 *  \n**About**  \nThe Brightcove Player is a high-performance video player that loads quickly, delivers high-quality adaptive bitrate HLS video across mobile platforms, and delivers seamless advertising and measurement capabilities.  \nThe Brightcove Native Player SDKs are built on top of native player frameworks. This allows you to take advantage of the speed, performance, and flexibility of the native OS, and extend it into a complete video experience.  \n[Click Here](https://www.brightcove.com/en/player) to learn more about the Brightcove Native Player.  \n**Key Features**  \n* VOD & LIVE HLS - High-quality, adaptive bitrate video supporting VOD playback, live streaming, player controls, and full-screen (portrait and landscape) and inline views.  \n* Advertising - Deliver pre-rolls, mid-rolls, and/or post-rolls to optimize your ad revenue. Ad insertion includes both VAST and VMAP support and is fully customizable on the item level. You can learn how to configure ads via the documentation [here](https://applicaster.zendesk.com/hc/en-us/articles/360023208871-Configuring-Video-Ads-in-Brightcove).  \n* Analytics - Fully integrated with Applicaster’s analytics plugin system so your Brightcove events will be tracked by any analytics plugins you choose. Relevant events can be found [here](https://docs.google.com/spreadsheets/d/1OfAQyb6cHvmqySzdFvXfBg-f73U0VwQIPg-YKNgdvcw/edit?usp=sharing).  \n* Error Handling - For optimal user experience when the sub-optimal occurs, we've provided error screens to explain to users what happened when a video cannot play, customizable via the configuration panel to the right.  \n**Pricing**  \nBrightcove Essentials starts at $12k a year. Contact your Account Manager for more details.  \n**Potential Future Features Include:**  \nClosed Captions (Required by law in some countries), TV Platform support, customizable player controls, and Chromecast support."
```

Comment:
The about text should not come from here, it should be uploaded as a md and refer to it when the plugin is published. If the plugin was published with the about text inside the code it needs to be changed.